kamailio/auth.cfg

```diff
@@ -30,18 +30,44 @@ route[AUTH_HEADERS]
 
 }
 
+route[AUTH_HEADERS_JSON]
+{
+   xavp_params_implode("hf", "$var(outx)");
+   $var(out) = $(var(outx){re.subst,/^(.*);$$/\1/});
+   $var(c) = $(var(out){param.count});
```

Comment:
@lazedo there is a lot of magic here, what is "c" and out vs outx, ect

Reply:
blame the output of  `xavp_params_implode`
`out` is a fix for `xavp_params_implode`
`c` is the count of  `params` imploded from xavp